apps/console/src/features/oidc-scopes/pages/oidc-scopes-edit.tsx

```diff
@@ -481,83 +484,86 @@ const OIDCScopesEditPage: FunctionComponent<RouteComponentProps<OIDCScopesEditPa
                                             readOnly={ isReadOnly }
                                         />
                                         { !isReadOnly &&
-                                            <Field.Button
+                                            (<Field.Button
```

Comment:
Shall we add the mandatory form id to the form here? Refer - https://github.com/wso2-enterprise/asgardeo-app-extensions/pull/2021

Reply:
Already added by https://github.com/wso2/identity-apps/commit/b92965577dd499eae7eb86273a8fd020fc53e269